lib/PuppeteerSharp.Tests/BrowserTests/Events/DisconnectedTests.cs

```diff
@@ -90,6 +90,7 @@ public async Task ShouldRejectWaitForSelectorWhenBrowserCloses()
                 //Whether from the Connection rejecting a message from the CDPSession 
                 //or from the CDPSession trying to send a message to a closed connection
                 Assert.IsType<TargetClosedException>(exception.InnerException);
+                Assert.Equal("Connection disposed", ((TargetClosedException)exception.InnerException).CloseReason);
```

Comment:
maybe write something like this?
```cs
Assert.Equal("Connection disposed", Assert.IsType<TargetClosedException>(exception.InnerException).CloseReason);
```

Reply:
IMHO I don't like the idea of having to asserts in the same line.